config/default.yml

```
@@ -543,6 +543,12 @@ RSpec/SingleArgumentMessageChain:
   VersionChanged: '1.10'
   StyleGuide: https://www.rubydoc.info/gems/rubocop-rspec/RuboCop/Cop/RSpec/SingleArgumentMessageChain
 
+RSpec/StubbedMock:
+  Description: Checks that message expectations do not have a configured response.
+  Enabled: true
```

I guess this has to be changed, right?
```suggestion
  Enabled: pending
```

---

Not necessarily. I guess we'll start adhering to that policy starting from 2.0. This wasn't the case for any of the 1.x releases at least.

---

Let's try to start adhering to the policy (our minimum rubocop version requirement already supports pending, doesn't it) - users will be happier

---

Agreed. At least we'll be able to see if pending works for extension cops.